### DISCUSSION TITLE
Book reference and printing

### DISCUSSION THREAD

**Discussion by Ouphix** [knowledgefutures/pubpub#1261] [Ideas]
Hello,

I would like to cite our EFPT psychotherapy guidebook in APA format, can you confirm that this is correct? 

Gargot, T., Koutsomitros, T., & Klotins, R. . EFPT Psychotherapy Guidebook, 1st edition, PubPub, 2020, https://epg.pubpub.org/

Maybe I should add the publisher? MIT press then?

Is it or will it be possible to compile a Pdf book for it? It can be much better for reading and sharing in some platforms for instance researchgate.

bw,

Thomas

**Comment by avanouse** [comment]
This is a great question—and one that interests me, because citation practices definitely have to evolve as new tools become available!

PubPub is not your publisher in the traditional sense—in the same way that Researchgate isn’t necessarily a publisher of your work, but what you’re using to share it. Here’s an example:
https://www.researchgate.net/publication/336065925_Aesthetic_Universals_in_Neil_Gaiman's_Post-Postmodern_Mythmaking

In this case, “Aesthetic Universals in Neil Gaiman’s Post-Postmodern Mythmaking” is published by _Prague Journal of English Studies_, not Researchgate. But sometimes it makes sense to bibliographers to include more or fewer points of information about the containers that contain the citational object to which they want to refer and direct you. How the compiler chooses to do this—in the same way that the second edition needs to be named, while the first edition doesn’t always—can be influenced by your choices to a certain extent. If you name PubPub, in the position within the citation that’s traditionally reserved for the publishing house—then it makes us happy to be named in affiliation with your work! But we’re not really your publisher: you are.

This is because what PubPub does for you (insofar as you are the creator of an independent community on PubPub) does not include many of the review services to which a traditional publishing house (the MIT Press, for example) would be subjecting your manuscript. Among its other benefits, producing your book independently means that the process of reviewing and styling chapter mss can be largely managed by your own team—as you know better than I do. While this is almost definitely the more efficient method, it does unfortunately mean that it would be dishonest for an independent user to claim an affiliation with the MIT Press for his, her, or their community.

The best overview of how MIT Press collaborates with PubPub on certain projects is here:
https://mitpressonpubpub.mitpress.mit.edu/pub/pathways

I hope this helps! It is possible, by the way, to submit a proposal for your open access project or book to be considered for publication by MIT Press. An intake form for such consideration is available [here](url).


Allison Vanouse
Editorial fellow, PubPub

**Reply by Ouphix** [reply]
dear allison,

thank you for your complete reply. 
Yes, it challenges a lot the traditional book method. 
I will reed the references you gave me thanks. However, I am sorry, the last one is not working.
bw,
Thomas

**Reply by avanouse** [reply]
Thomas, I should have replied to this months ago, I'm sorry! Here is the link: https://docs.google.com/forms/d/e/1FAIpQLSfdrR95V69UHG31mQDVTwWj-_UX6MqEO2eOCf1ToPKn7ecsmQ/viewform
